docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
+      - my_media:/app/files/media
+      - my_static:/app/staticfiles
+    depends_on:
+      - db
```

Comment:
The `depends_on` option does not wait for `db` to be ready before starting `app`. Consider using a script or tool to check the database readiness, as `depends_on` only ensures that the `db` service is started, not that it is ready to accept connections.